app/code/Magento/Inventory/Test/Acceptance/Test/AdminUpdateSourceDataTest.xml

```diff
@@ -0,0 +1,90 @@
+
+        <conditionalClick selector="{{AdminEditSourceCarriersSection.closed}}" dependentSelector="{{AdminEditSourceCarriersSection.opened}}" visible="false" stepKey="clickOnCarriers1"/>
+        <checkOption selector="{{AdminEditSourceCarriersSection.useGlobalShippingConfiguration}}" stepKey="checkUseGlobalShippingConfiguration1"/>
+
+        <click selector="{{AdminProductFormActionSection.saveButton}}" stepKey="clickOnSaveContinue1"/>
+
+        <fillField userInput="{{TestSource2.name}}" selector="{{AdminEditSourceGeneralSection.name}}" stepKey="fillNameField2"/>
+        <checkOption selector="{{AdminEditSourceGeneralSection.isEnabled}}" stepKey="checkIsEnabled2"/>
+        <fillField userInput="{{TestSource2.description}}" selector="{{AdminEditSourceGeneralSection.description}}" stepKey="fillDescriptionField2"/>
+
+        <!--Todo: MQE-677 needs to be merged into Develop before we can use "0" in the Data. Hardcoded "0" for now.-->
+        <fillField userInput="0" selector="{{AdminEditSourceGeneralSection.latitude}}" stepKey="fillLatitudeField2"/>
+        <fillField userInput="0" selector="{{AdminEditSourceGeneralSection.longitude}}" stepKey="fillLongitudeField2"/>
+
+        <conditionalClick selector="{{AdminEditSourceContactInfoSection.closed}}" dependentSelector="{{AdminEditSourceContactInfoSection.opened}}" visible="false" stepKey="clickOnContactInfo2"/>
+        <fillField userInput="{{TestSource2.contact_name}}" selector="{{AdminEditSourceContactInfoSection.contactName}}" stepKey="fillContactNameField2"/>
+        <fillField userInput="{{TestSource2.email}}" selector="{{AdminEditSourceContactInfoSection.email}}" stepKey="fillEmailField2"/>
+        <fillField userInput="{{TestSource2.phone}}" selector="{{AdminEditSourceContactInfoSection.phone}}" stepKey="fillPhoneField2"/>
+        <fillField userInput="{{TestSource2.fax}}" selector="{{AdminEditSourceContactInfoSection.fax}}" stepKey="fillFaxField2"/>
+
+        <conditionalClick selector="{{AdminEditSourceAddressDataSection.closed}}" dependentSelector="{{AdminEditSourceAddressDataSection.opened}}" visible="false" stepKey="clickOnAddresses2"/>
+        <selectOption userInput="{{TestSource2.country}}" selector="{{AdminEditSourceAddressDataSection.country}}" stepKey="selectCountry2"/>
+        <selectOption userInput="{{TestSource2.stateProvince}}" selector="{{AdminEditSourceAddressDataSection.state}}" stepKey="selectState2"/>
+        <fillField userInput="{{TestSource2.city}}" selector="{{AdminEditSourceAddressDataSection.city}}" stepKey="fillCityField2"/>
+        <fillField userInput="{{TestSource2.street}}" selector="{{AdminEditSourceAddressDataSection.street}}" stepKey="fillStreetField2"/>
+        <fillField userInput="{{TestSource2.postcode}}" selector="{{AdminEditSourceAddressDataSection.postcode}}" stepKey="fillPostcodeField2"/>
+
+        <actionGroup ref="AdminSaveAndClose" stepKey="saveAndClose1"/>
```

Comment:
In test scenario I see we should click "Save and Continue" and verify data in all tabs is correct.

```
18. click button "Save & Continue"
19. verify that data in all tabs is correct
20. Then verify in Manage Sources grid that Source's data changes applied properly
```

Reply:
Test updated.